### PR TITLE
FAI-13445: Have all converters writing geo_location use shared utility

### PR DESCRIPTION
**Pull request by chalenge**
## Description

- Have all geo location converters use shared utility

## Type of change
- [ ] Bug fix
- [X] New feature
- [ ] Breaking change

### CODE REVIEW

File: destinations/airbyte-faros-destination/test/converters/__snapshots__/bamboohr.test.ts.snap [hunk 1]
```diff
@@ -4,8 +4,8 @@ exports[`bamboohr process records from all streams 1`] = `
 Array [
   "Processed 87 records",
   "Processed records by stream: {\\\\\\"mytestsource__bamboohr__users\\\\\\":87}\\"},\\"type\\":\\"LOG\\"}",
-  "Would write 493 records",
-  "Would write records by model: {\\\\\\"geo_Address\\\\\\":87,\\\\\\"geo_Location\\\\\\":87,\\\\\\"identity_Identity\\\\\\":87,\\\\\\"org_Department\\\\\\":9,\\\\\\"org_Employee\\\\\\":87,\\\\\\"org_Team\\\\\\":26,\\\\\\"org_TeamMembership\\\\\\":110}\\"},\\"type\\":\\"LOG\\"}",
+  "Would write 371 records",
```
Comment (chalenge) [inline destinations/airbyte-faros-destination/test/converters/__snapshots__/bamboohr.test.ts.snap:7]: we were writing duplicate records hence reduction

**Review by ypc-faros — APPROVED** [review]
💥

**Comment by ypc-faros** [comment]
There's a failing test that checks the dest spec. You're adding to it so that's expected. Just need to check whether the spec looks good (`./bin/main spec | jq` from `destinations/airbyte-faros-destination`) and update the test snapshot

**Comment by chalenge** [comment]
> There's a failing test that checks the dest spec. You're adding to it so that's expected. Just need to check whether the spec looks good (`./bin/main spec | jq` from `destinations/airbyte-faros-destination`) and update the test snapshot

I always forget this test 😫

**Pull request merged**